diff --git a/node_modules/minecraft-protocol/src/client/microsoftAuth.js b/node_modules/minecraft-protocol/src/client/microsoftAuth.js
index 8aea989..40cab1e 100644
--- a/node_modules/minecraft-protocol/src/client/microsoftAuth.js
+++ b/node_modules/minecraft-protocol/src/client/microsoftAuth.js
@@ -19,7 +19,7 @@ async function authenticate (client, options) {
   validateOptions(options)
 
   if (!client.authflow) client.authflow = new PrismarineAuth(options.username, options.profilesFolder, options, options.onMsaCode)
-  const { token, entitlements, profile, certificates } = await client.authflow.getMinecraftJavaToken({ fetchProfile: true, fetchCertificates: !options.disableChatSigning }).catch(e => {
+  const { token, entitlements, profile, certificates } = await client.authflow.getMinecraftJavaToken({ fetchProfile: true, fetchCertificates: !options.disableChatSigning, fetchEntitlements: true }).catch(e => {
     if (options.password) console.warn('Sign in failed, try removing the password field\n')
     if (e.toString().includes('Not Found')) console.warn(`Please verify that the account ${options.username} owns Minecraft\n`)
     throw e
